sdk/openai/openai/src/rest/openAIClient.ts

```diff
@@ -20,7 +20,7 @@ export default function createClient(
 ): OpenAIContext {
   const baseUrl = options.baseUrl ?? `${endpoint}/openai`;
   options.apiVersion = options.apiVersion ?? "2024-03-01-preview";
-  const userAgentInfo = `azsdk-js-openai-rest/1.0.0-beta.12`;
+  const userAgentInfo = `azsdk-js-openai-rest/1.0.0-beta.13`;
```

Comment:
Just a question Is there someplace we know that this version is associated with deprecation (other than this PR?)

Reply:
This version is not released yet. I am talking to Timo to have the deprecation announcement in the release blog as well.